src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -188,4 +190,38 @@
    :margin-top       2
    :margin-bottom    4
    :margin-right     2
-   :background-color :white})
\ No newline at end of file
+   :background-color :white})
+
+(def public-chats-item-container
+  {:flex-direction :row
+   :padding        3})
+
+(def public-chats-icon-container
+  {:flex 0.15})
+
+(defn public-chats-icon [color]
+  {:width            50
+   :height           50
+   :border-radius    25
+   :background-color color
+   :flex-direction   :row
+   :justify-content  :center
+   :align-items      :center})
+
+(def public-chats-icon-text
+  {:font-size 25
+   :color     :gray})
```

Comment:
Are you sure this is the correct gray? We have a bunch of custom ones.

Reply:
I'm not, but I'm not sure about any of the custom ones either. I can replace it with one of them now, sure, but this will still remain a subject to change in the pixel-good PR later. It makes little sense to eyeball-compare a feature complete design with a partial implementation, IMO.